python/MooseDocs/test/content/extensions/alert.md

```diff
@@ -31,5 +34,5 @@ int x;
 !alert-end!
 
 
-!alert note title=[Google](https://google.com)
+!alert tip title=[Google](https://google.com)
```

Comment:
Is there still a `!alert note` somewhere?

Reply:
Yes, all the other auxiliary examples used `!alert note`, so I wanted to try something different for the title with link demo